grails-app/conf/Config.groovy

```diff
@@ -350,12 +380,19 @@ portal {
tion%2CassociatedMedia%2CassociatedReferences%2CassociatedSequences%2CassociatedTaxa%2CotherCatalogNumbers%2CoccurrenceRemarks%2CorganismID%2CorganismName%2CorganismScope%2CassociatedOccurrences.p%2CassociatedOrganisms%2CpreviousIdentifications%2CorganismRemarks%2CmaterialSampleID%2CeventID%2CparentEventID%2CfieldNumber%2CeventDate.p%2CeventTime%2CstartDayOfYear%2CendDayOfYear%2Cyear.p%2Cmonth.p%2Cday.p%2CverbatimEventDate%2Chabitat.p%2CsamplingProtocol.p%2CsamplingEffort%2CsampleSizeUnit%2CfieldNotes%2CeventRemarks%2ClocationId%2ChigherGeographyID%2ChigherGeography%2Ccontinent%2CwaterBody%2CislandGroup%2Cisland%2Ccountry.p%2CcountryCode%2CstateProvince.p%2Ccounty%2Cmunicipality%2Clocality.p%2CverbatimLocality%2CminimumElevationInMeters.p%2CmaximumElevationInMeters.p%2CverbatimElevation.p%2CminimumDepthInMeters.p%2CmaximumDepthInMeters.p%2CverbatimDepth.p%2CminimumDistanceAboveSurfaceInMeters%2CmaximumDistanceAboveSurfaceInMeters%2ClocationAccordingTo%2ClocationRemarks%2CdecimalLatitude.p%2CdecimalLongitude.p%2CgeodeticDatum.p%2CcoordinateUncertaintyInMeters.p%2CcoordinatePrecision.p%2CpointRadiusSpatialFit%2CverbatimCoordinates%2CverbatimLatitude%2CverbatimLongitude%2CverbatimCoordinateSystem%2CverbatimSRS%2CfootprintWKT%2CfootprintSRS%2CfootprintSpatialFit%2CgeoreferencedBy.p%2CgeoreferencedDate.p%2CgeoreferenceProtocol.p%2CgeoreferenceSources.p%2CgeoreferenceVerificationStatus.p%2CgeoreferenceRemarks%2CgeologicalContextID%2CearliestEonOrLowestEonothem%2ClatestEonOrHighestEonothem%2CearliestEraOrLowestErathem%2ClatestEraOrHighestErathem%2CearliestPeriodOrLowestSystem%2ClatestPeriodOrHighestSystem%2CearliestEpochOrLowestSeries%2ClatestEpochOrHighestSeries%2CearliestAgeOrLowestStage%2ClatestAgeOrHighestStage%2ClowestBiostratigraphicZone%2ChighestBiostratigraphicZone%2ClithostratigraphicTerms%2Cgroup%2Cformation%2Cmember%2Cbed%2CidentificationID%2CidentificationQualifier%2CtypeStatus.p%2CidentifiedBy.p%2CdateIdentified.p%2CidentificationReferences.p%2CidentificationVerificationStatus%2CidentificationRemarks%2CtaxonID%2CscientificNameID%2CacceptedNameUsageID%2CparentNameUsageID%2CoriginalNameUsageID%2CnameAccordingToID%2CnamePublishedInID%2CtaxonConceptID.p%2CscientificName.p%2CacceptedNameUsage%2CparentNameUsage%2CoriginalNameUsage%2CnameAccordingTo.p%2CnamePublishedIn%2CnamePublishedInYear%2ChigherClassification%2Ckingdom.p%2Cphylum.p%2Cclasss.p%2Cclasss%2Corder.p%2Cfamily.p%2Cgenus.p%2Csubgenus%2Cspecies.p%2CspecificEpithet%2CinfraspecificEpithet%2CtaxonRank.p%2CverbatimTaxonRank%2CscientificNameAuthorship%2CvernacularName.p%2CnomenclaturalCode.p%2CtaxonomicStatus%2CnomenclaturalStatus%2CtaxonRemarks%2CmeasurementDeterminedDate%2CrelationshipAccordingTo%2CresourceRelationshipID%2CmeasurementRemarks%2CmeasurementValue%2CrelationshipRemarks%2CresourceID%2CrelationshipEstablishedDate%2CrelationshipOfResource%2CmeasurementMethod%2CmeasurementID%2CmeasurementType%2CmeasurementUnit%2CmeasurementDeterminedBy%2CmeasurementAccuracy%2CrelatedResourceID"*/
```

Comment:
Its the fields that Seb was after, but it is way to much to specify and creates a download file that is full of empty attributes. It remains here until it can be removed/reduced/modded It was tedius to assemble :) Ok'd by @pblain for now